dynawo/sources/Solvers/AlgebraicSolvers/DYNSolverKINAlgRestoration.cpp

```diff
@@ -36,8 +36,8 @@
 #include "DYNSolverCommon.h"
 #include "DYNTrace.h"
 #include "DYNMacrosMessage.h"
-#include "DYNModel.h"
-#include "DYNSparseMatrix.h"
+#include "DYNSubModel.h"
```

Comment:
This is completely forbidden, we need to find another way of doing this

Reply:
artefact from previous implementation try and can be removed without modifying logic.

Reply:
is it the same for the CMakeList?

Reply:
Correct it has to be reverted